examples/simulation/Tutorial1/src/FairTutorialDet1.cxx

```diff
@@ -165,6 +165,15 @@ void FairTutorialDet1::Reset()
   fFairTutorialDet1PointCollection->Clear();
 }
 
+Bool_t FairTutorialDet1::CheckIfSensitive(std::string name)
```

Comment:
```suggestion
Bool_t FairTutorialDet1::CheckIfSensitive(const std::string& name)
{
  if (name.find("tutdet") != std::string::npos) {
    return kTRUE;
  }
  return kFALSE;
}
```
to avoid string copy/allocation.

Reply:
or even just `return name.find("tutdet") != std::string::npos;` if `bool` converts to `Bool_t`